components/http_foundation/introduction.rst

```diff
@@ -483,6 +483,9 @@ abstracts the hard work behind a simple API::
 
     $response->headers->set('Content-Disposition', $d);
 
+You can add the file's content as usual, for example by using
```

Comment:
I'm actually wondering why we don't set the content as the constructor argument, as done most of the time when setting response content?